brax/envs/__init__.py

```diff
@@ -30,33 +30,79 @@
 from brax.envs import ur5e
 from brax.envs import wrappers
 
-_envs = {
-    'fetch': fetch.Fetch,
-    'ant': ant.Ant,
-    'grasp': grasp.Grasp,
-    'halfcheetah': halfcheetah.Halfcheetah,
-    'humanoid': humanoid.Humanoid,
-    'ur5e': ur5e.Ur5e,
-    'reacher': reacher.Reacher,
-    'reacherangle': reacherangle.ReacherAngle,
+_envs: Dict[str, Type[env.Env]] = {
```

Comment:
To be consistent with the rest of the library, please avoid type annotations for internal constants.